Dshop/apps/payments/tests/test_checkout_session.py

```diff
@@ -0,0 +1,14 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.urls import reverse
+
+
+class PaymentTestCase(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username='testuser', password='testpassword')
+
+    def test_create_checkout_session(self):
+        self.client.login(username='testuser', password='testpassword')
+        response = self.client.get(reverse('create_checkout_session'))
```

Comment:
Nice. In order to avoid real call to stripe please use restes.mock. 
https://pypi.org/project/requests-mock/
1. Test should be independent from external resource. Imagine somebody is working on a train with lousy connection, or on a plane - test will fail because no network. 
2. This will also fail if stripe has issues. Or you will remove the key.

Reply:
I made changes according to your suggestions, but I'm not sure if it's ok.